style.css

```diff
@@ -0,0 +1,250 @@
+@import url('https://fonts.googleapis.com/css2?family=Poppins:ital,wght@0,100;0,200;0,300;0,400;0,500;0,600;0,700;0,800;0,900;1,100;1,200;1,300;1,400;1,500;1,600;1,700;1,800;1,900&display=swap');
+
+* {
+  padding: 0;
+  margin: 0;
+}
+
+html {
+  font-family: 'Poppins', sans-serif;
+}
+
+body {
+  position: absolute;
+  background-color: #e5e5e5;
+}
+
+header {
+  color: #6070ff;
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+  padding: 1em 7%;
+  background-color: #fff;
+}
+
+.logo > a {
+  text-decoration: none;
+  color: inherit;
+}
+
+.headline-main {
+  max-height: 800px;
+  min-height: 100vh;
+  width: auto;
+  display: flex;
+  justify-content: center;
+  align-items: center;
+  padding: 0 7%;
+  background-image: url(./assets/header-shape.png);
+  background-color: #fff;
+  background-repeat: no-repeat;
+  background-position: 50%;
+}
+
+.headline {
+  margin: 10px 0;
+  display: flex;
+  flex-direction: column;
+  align-items: flex-start;
+  height: 100%;
+}
+
+.headline-heading {
+  margin: 10px 0;
+  color: #172b4d;
+  font-weight: 700;
+  font-size: 2.5rem;
+  line-height: 52px;
+  display: flex;
+}
+
+.headline-paragraph {
+  margin: 10px 0;
+  color: #344563;
+  font-family: 'Poppins', sans-serif;
+  font-style: normal;
+  font-weight: 400;
+  font-size: 16px;
+  line-height: 24px;
+}
+
+.headline-sm-suport {
+  margin: 10px 0;
+  color: #7f8cff;
+  font-weight: 500;
+}
+
+.social-icons {
+  width: 25%;
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+  color: #505f79;
+  list-style: none;
+}
+
+.social-icons > a,
+i {
+  text-decoration: none;
+  color: #505f79;
+}
+
+/* Works Section Styling Starts */
+.works-card {
+  color: #172b4d;
+  background-color: #fff;
+  border: 1px solid #dfe1e6;
+  border-radius: 1em;
+  display: grid;
+  flex-direction: column;
+  align-items: flex-start;
+  justify-content: center;
```

Comment:
- [ ] You have done a great job displaying the cards using `grid`. However, the properties used to position the cards are not met for gride. You have instead used the `flexbox` properties. Kindly refer to this documentation to learn more about [grid properties](https://www.w3schools.com/cssref/pr_grid.php) and use them to position the section ☺️ . This will help us to have a clean code that is easy to maintain 👍 .

Reply:
I updated grid to flexbox.